code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -74,10 +76,48 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/mountain/Move()
+	var/turf/floor = get_turf(src)
+	for(var/thing in floor)
+		if(istype(thing, /obj/effect/decal/cleanable/blood))
```

Comment:
I'd suggest it only eats specific gibs, only those created by dead humans.
Remember that Rudolta exists...

Reply:
Thats supposed to be the issue. Filthy areas are supposed to make him more dangerous.